src/cocaine-app/infrastructure_cache.py

```diff
@@ -1,7 +1,6 @@
 import logging
 import socket
 import time
-import traceback
```

Comment:
Because it was unused in the module.
This should have been a part of another pool request. Though it was a separate `Pepify` commit.